src/aiai_eval/task.py

```diff
@@ -1,11 +1,39 @@
 """Abstract Task class."""
+import logging
+import random
+import subprocess
+import warnings
+from abc import ABC, abstractmethod
+from functools import partial
+from subprocess import CalledProcessError
+from typing import Any, Dict, Optional, Sequence, Union
 
-from abc import ABC
-from typing import Dict
+import numpy as np
+import torch
+import torch.nn as nn
+from datasets import Dataset, DatasetDict, load_dataset, load_metric
+from torch.utils.data import DataLoader
+from tqdm import tqdm
```

Comment:
This ensures that `tqdm` progress bars also work in notebooks.

```suggestion
from tqdm.auto import tqdm
```